flake.nix

```diff
@@ -29,6 +29,17 @@
           openroad = (nixpkgs.lib.callPackageWith pkgs) ./default.nix {
             flake = self;
           };
+          openroad-release = all.openroad.overrideAttrs (fa: pa: rec {
+            pname = "openroad-release";
+            version = "24Q3";
+            src = pkgs.fetchFromGitHub {
+              owner = "The-OpenROAD-Project";
+              repo = "OpenROAD";
+              rev = version;
+              fetchSubmodules = true;
+              sha256 = "sha256-Ye9XJcoUxtg031eazT4qrexvyN0jZHd8/kmvAr/lPzk=";
+            };
+          });
           default = all.openroad;
```

Comment:
I didn't change the default because I didn't want to disrupt anybody's flow, but I agree that it would make sense. In any case it's still possible to use all nix commands by adding `#openroad-release`.